ui/desktop/app/controllers/scopes/scope/projects/settings/index.js

```diff
@@ -18,4 +18,12 @@ export default class ScopesScopeProjectsSettingsIndexController extends Controll
   toggleTheme({ target: { value: theme } }) {
     return this.application.toggleTheme(theme);
   }
+
+  /**
+   * Call Application controller's signoutAttempt method
+   */
+  @action
+  signoutAttempt() {
```

Comment:
Mentioned elsewhere but I don't think this is necessary as you can keep the original `invalidateSession` function